{"uid":"9ba905f6623a7998","name":"test_deploy_async_replication_setup","fullName":"tests.integration.ha_tests.test_async_replication#test_deploy_async_replication_setup","historyId":"d78aa2d7506528b71f817b6a92e55938","time":{"start":1717897828553,"stop":1717899850391,"duration":2021838},"description":"Build and deploy two PostgreSQL cluster in two separate models to test async replication.","descriptionHtml":"<p>Build and deploy two PostgreSQL cluster in two separate models to test async replication.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent\n  postgresql-test-app/0 [allocating] waiting: installing agent","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/tests/integration/ha_tests/test_async_replication.py\", line 113, in test_deploy_async_replication_setup\n    await gather(\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2969, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent\n  postgresql-test-app/0 [allocating] waiting: installing agent","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1717897827712,"stop":1717897827712,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1717897827712,"stop":1717897827712,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1717897827712,"stop":1717897828061,"duration":349},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"first_model","time":{"start":1717897828061,"stop":1717897828061,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"second_model","time":{"start":1717897828062,"stop":1717897828552,"duration":490},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1717897828553,"stop":1717897828553,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy two PostgreSQL cluster in two separate models to test async replication.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent\n  postgresql-test-app/0 [allocating] waiting: installing agent","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/tests/integration/ha_tests/test_async_replication.py\", line 113, in test_deploy_async_replication_setup\n    await gather(\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2969, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent\n  postgresql-test-app/0 [allocating] waiting: installing agent","steps":[],"attachments":[{"uid":"143b6d6649852ccd","name":"log","source":"143b6d6649852ccd.txt","type":"text/plain","size":23028}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1717899850449,"stop":1717899850449,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"second_model::finalizer","time":{"start":1717899850467,"stop":1717899850485,"duration":18},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1717899850486,"stop":1717899850987,"duration":501},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1717899850988,"stop":1717899850988,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1717899850988,"stop":1717899850988,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1717899850988,"stop":1717899850988,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"@pytest.mark.skipif(False, reason='Requires juju 3')"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_async_replication"},{"name":"host","value":"runner"},{"name":"thread","value":"15267-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_async_replication"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"8e252d39f8d0b7b0","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"96019541efb8b516","reportUrl":"../72//#testresult/96019541efb8b516","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent\n  postgresql-k8s/1 [allocating] waiting: installing agent\n  postgresql-k8s/2 [allocating] waiting: installing agent\n  postgresql-test-app/0 [allocating] waiting: installing agent","time":{"start":1717813314695,"stop":1717815336398,"duration":2021703}}]},"tags":["asyncio","abort_on_fail","@pytest.mark.skipif(False, reason='Requires juju 3')","group(1)"]},"source":"9ba905f6623a7998.json","parameterValues":[]}